src/app/core/entity/schema/default-value-config.ts

```diff
@@ -41,5 +41,6 @@ export type DefaultValueMode =
 export interface AutomatedConfigRule {
   relatedEntity: string;
   relatedField: string;
+  mappedProperty: string;
```

Comment:
show dropdown in admin UI to allow user to select (if multiple fields link to the entity type)